components/contact/contact.ts

```diff
@@ -10,6 +10,11 @@ class Contact {
     );
   }
 
+  init(): void {
+    this.onClick();
+    this.submitForm();
+  }
+
   removeRevealClasses(): void {
     this.contactForms_.map((form, formIndex) => {
```

Comment:
map is the wrong function to use here. You use map when you are converting one array of elements to a new array of modified version of those elements.

In this case you're just doing something with each element, so the right function to use is forEach.